File: 프로그래머스/0/120849. 모음 제거/모음 제거.py
def solution(my_string):
    answer = ''
    for i in range(len(my_string)):
        if my_string[i]!=('a'):
            if my_string[i]!=('e'):
                if my_string[i]!=('i'):
                    if my_string[i]!=('o'):
                        if my_string[i]!=('u'):
                            answer+=my_string[i]          
    return answer

File: 프로그래머스/0/120849. 모음 제거/모음 제거.java
class Solution {
    public String solution(String my_string) {
        String answer = "";
        String[] A = {"a", "e", "i", "o", "u"};

        for (int i = 0; i < my_string.length(); i++) {
            boolean isVowel = false;

            for (int j = 0; j < A.length; j++) {
                if (my_string.charAt(i) == A[j].charAt(0)) {
                    isVowel = true;
                    break;
                }
            }

            if (!isVowel) {
                answer += my_string.charAt(i);
            }
        }

        return answer;
    }
}